### ISSUE TITLE
Add-On-Befragung

### ISSUE THREAD

**Issue by andreaschu** [dzhw/studienberechtigte_2018.3b#209]
ist verfügbar unter: https://presentation.dzhw.eu/SLC_2018_3b-addon

**Comment by heikQ** [comment]
okies, passt von meiner Seite aus so, danke. Hier zu machen?

**Comment by AliceFoelker** [comment]
@andreaschu ist das ein technischer Fehler, dass mit das über die Simulation als Karussell auf mobil angezeigt wird?
![grafik](https://user-images.githubusercontent.com/73596957/105360021-6a65b700-5bf8-11eb-8ea3-65c598adbe5e.png)

**Comment by andreaschu** [comment]
@AliceFoelker nein, das ist eine offene Matrix, damit es auf dem großen Bildschirm tabellarisch angeordnet ist und nicht als einzelne offene Fragen untereinander.

**Comment by andreaschu** [comment]
@dzhwmeisner können wir in der Add-On-Befragung das Carousel deaktivieren?

**Comment by AliceFoelker** [comment]
@andreaschu wenn nicht, könnten wir einfach die jeweilien Strings wie in der Desktop-Version stehen lassen und die Beschriftungen, also Name, Ort etc. drüber schreiben?

**Comment by dzhwmeisner** [comment]
Im nächsten Rollout verfügbar

**Comment by andreaschu** [comment]
Ist ausgerollt und kann getestet werden.

**Comment by AliceFoelker** [comment]
@andreaschu Der Text im Add on ist hier dann quasi doppelt. Stand das auch in der vorherigen Befragung noch mal über der Adressaktualisierung? Wenn es so stehen bleiben soll, dann würde ich das nur in den Klammern entfernen mit dem Jahr.
![grafik](https://user-images.githubusercontent.com/73596957/105497981-e119b780-5cbf-11eb-98ba-e1a2dde10d71.png)

![grafik](https://user-images.githubusercontent.com/73596957/105497925-ccd5ba80-5cbf-11eb-90b2-c2d6c4babd8b.png)

**Comment by andreaschu** [comment]
Du meinst auf der letzten Seite der Hauptbefragung und auf der ersten Seite der Add-On? Ja, das war letztes Mal genauso.

**Comment by AliceFoelker** [comment]
@andreaschu oki, dann würde ich lediglich das in der Klammer löschen, oder? @jocarstensen

**Comment by andreaschu** [comment]
@AliceFoelker ich habe '(in einem Jahr)' gelöscht und die Befragung aktualisiert.

**Comment by AliceFoelker** [comment]
@andreaschu sollte die Add-On Befragung in der test-local-version nun eigentlich schon zu erreichen sein? Wenn ich auf den Link klicke erscheint nur das:
![grafik](https://user-images.githubusercontent.com/73596957/105839408-b8503580-5fd1-11eb-86fa-f65ed078ad9b.png)

**Comment by andreaschu** [comment]
@AliceFoelker nein, der Übergang zur Adressabfrage geht nur in der Produktiversion, weil in der Testversion keine Preloadtoken für die Add-On-Befragung in der Datenbank sind. Der Link ist aber der richtige und bereits produktive Link (survey01.dzhw.eu...).
